README.rst

```diff
@@ -151,12 +146,16 @@ Running unit tests::
 
 Running integration tests::
 
+    # fetch-kafka needed only once
+    make fetch-kafka
     make integrationtest
 
-The tests can be run in an IDE too, gotchas:
+When running tests in an IDE, ensure that the working directory is set to Git root, e.g. in PyCharm you can create a configuration template with the correct Working directory.
+
+It's recommended to install :code:`pre-commit` GitHub. There are several coding style checks in `GitHub Actions <https://github.com/aiven/karapace/actions>`_,
```

Comment:
```suggestion
It's recommended to install `pre-commit`<https://pre-commit.com/>. There are several coding style checks in `GitHub Actions <https://github.com/aiven/karapace/actions>`_,
```

Maybe direct link to precommit tool would serve better?

Reply:
I clarified the instructions.